code/game/objects/items/devices/chameleonproj.dm

```diff
@@ -49,9 +49,17 @@
 		return
 	if(istype(target, /obj/structure/falsewall))
 		return
+	if(target.alpha == 0)
```

Comment:
I'd go with != 255 since there probably exists things with alpha 1 or similar "not invisible honest"

Reply:
What you said in the comment.